srcpkgs/linux5.5/files/x86_64-dotconfig

```diff
@@ -22,7 +22,7 @@ CONFIG_THREAD_INFO_IN_TASK=y
 #
 CONFIG_INIT_ENV_ARG_LIMIT=32
 # CONFIG_COMPILE_TEST is not set
-CONFIG_LOCALVERSION=""
+CONFIG_LOCALVERSION="_1"
```

Comment:
This one does not seem to have effect, but the right one is to make it void `=""`